test/testdrive/github-7290.td

```diff
@@ -24,7 +24,7 @@ bar
   )
   FORMAT BYTES;
 
-> SELECT data FROM posix ORDER BY mz_record;
```

Comment:
Could this become flaky without the ordering? There are other instances of this below.